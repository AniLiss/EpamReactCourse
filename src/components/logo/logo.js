import React from 'react';
import './logo.scss';

export const Logo = () => {
    return (
        <h1 className="logo">netflixroulette</h1>
    )
};
