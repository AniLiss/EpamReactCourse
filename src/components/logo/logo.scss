.logo {
  color: #ff2f5b;
  font-size: 18px;
}
