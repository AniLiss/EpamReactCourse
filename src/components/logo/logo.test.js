import {shallow} from 'enzyme';
import React from 'react';
import {Logo} from './logo';

describe('Logo', () => {
    it('is expected to render logo correct', () => {
        const wrapper = shallow(<Logo />);
        expect(wrapper).toMatchSnapshot();
    })
});
