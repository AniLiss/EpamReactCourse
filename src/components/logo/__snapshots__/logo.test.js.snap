// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo is expected to render logo correct 1`] = `
<h1
  className="logo"
>
  netflixroulette
</h1>
`;
